src/project-generator.ts

```diff
@@ -15,7 +15,7 @@
 import { StatusCodes } from 'http-status-codes';
 import { CodeConverter, CodeConversionResult } from './code-converter';
 import { MS_TO_WAIT_FOR_GITHUB, LOCAL_VSPEC_PATH, APP_MANIFEST_PATH, MAIN_PY_PATH } from './utils/constants';
-import { DataPointDefinition, decode, delay, encode, isNewAppManifest } from './utils/helpers';
+import { DataPointDefinition, NewAppManifest, decode, delay, encode } from './utils/helpers';
```

Comment:
since we don't have any oldAppManifest here, this should just be called AppManifest
```suggestion
import { DataPointDefinition, AppManifest, decode, delay, encode } from './utils/helpers';
```